definitions/procedures/upgrade_sync_plans_disable.rb

```diff
@@ -0,0 +1,34 @@
+class Procedures::UpgradeSyncPlansDisable < ForemanMaintain::Procedure
```

Comment:
The checks and procedures should follow the same namespacing strategy, if we have `Checks::SyncPlan`, we should have `Procedures::SyncPlans` where we would contain the code regarding this